### PR TITLE
filter by online and prepare template for screenshot

### LINKED ISSUE #18: The "remove team" button doesn't seem to do anything
I am unable to "remove team" -  the button doesn't seem to do anything.

jamstack_hackathon_2018___freecodecamp_org

### PR DESCRIPTION
**Pull request by tbushman**
filter teams by isOnlineHackathon and placeholder for screenshot.  
Closes #18

### CODE REVIEW

File: server/team/team.route.js [hunk 1]
```diff
@@ -20,6 +20,10 @@ router
   .get(teamCtrl.single)
   .post(ifNoUserRedirect(), ifNoBody400, validate(validators.createTeam), teamCtrl.update);
 
+router
+  .route('/analyze/:teamId')
+  .get(teamCtrl.analyze);
```
Comment (tbushman) [inline server/team/team.route.js:30]: this is so I can populate the table in dev

File: server/views/partials/leaderboard.pug [hunk 2]
```diff
@@ -9,21 +14,15 @@ table.table(cellspacing="0px", width="100%", style="width: 100%;overflow:hidden;
       th Online-only
       th Lighthouse
       th Collaborators
+      th Screenshot
       // th(v-for='(key, j) in Object.keys(teams[0])', v-if="['_id', '__v'].indexOf(key) === -1" v-text='key', scope="col")
-  tbody
+
+
+  tbody(v-if="!checked")
```
Comment (tbushman) [inline server/views/partials/leaderboard.pug]: wrong ... will fix

**Comment by tbushman** [comment]
Don't merge yet, please. Need to fix a couple of things.

**Comment by QuincyLarson** [comment]
@tbushman OK - we will wait for you. Thanks again for helping build this leaderboard! It will immediately have a lot of people using it.

**Comment by tbushman** [comment]
@QuincyLarson Thank you. I would need this merged so that @scissorsneedfoodtoo can work off of the latest, but it would need someone to test it locally. This will close #18 and add viewWebhooks to the manageTeam view.

**Comment by scissorsneedfoodtoo** [comment]
This looks great! Just noticed one thing with the screenshot on the leaderboard.

If I make a team, fill out all the info, then check the leaderboard everything looks fine. But if I delete my team and check the leaderboard, the screenshot is still there:

image

Is that something you can fix in this PR? Or would you rather leave it for another PR?

File: server/views/partials/team.pug [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  ul(v-else-if="key === 'collaborators'")
+    li.avatar(v-for="(member, k) in team[key]")
+      img(:src="`https://github.com/${member}.png`", :title="member")
+  span(v-else-if="key === 'lighthouse'")
+    ul(v-if="team[key].length", v-for="(result, l) in team[key]")
+      li(v-for="(metric, m) in Object.keys(result)", v-if="metric !== '_id'")
+        strong(v-text="metric+': '")
+        span(v-if="metric === 'date'", v-text="new Date(result[metric]).getFullYear() + '/'+ new Date(result[metric]).getMonth()+'/'+ new Date(result[metric]).getDate()+' '+ new Date(result[metric]).getHours()+':'+ new Date(result[metric]).getMinutes()")
+        span(v-else-if="metric !== 'date'", v-text="result[metric]")
+td(v-if="team && !team.screenshot")
```
Comment (tbushman) [inline server/views/partials/team.pug:17]: @scissorsneedfoodtoo OK, this should fix that.

**Comment by scissorsneedfoodtoo** [comment]
Everything LGTM!

**Comment by tbushman** [comment]
@scissorsneedfoodtoo Many thanks!

**Pull request merged**